tools/marvin/marvin/config/test_data.py

```diff
@@ -1881,29 +1912,181 @@
                 "name": "windowsxdtemplate",
                 "passwordenabled": False,
                 "ostype": "Windows 8 (64-bit)"
-            },
-        },
+            }
+        }
 
+    "configurableData":
+        {
```

Comment:
It seems this PR broke test_data.py https://github.com/apache/cloudstack/actions/runs/5186024194/jobs/9346611059#step:12:104 cc @Pearl1594 @weizhouapache @rohityadavcloud @DaanHoogland 

Failures were not seen earlier in the Trillian test runs as that uses its own test_data. I suggest we revert file and add only changes for `vpc_vpn` key ln 1942-2089?